badge.css

```diff
@@ -0,0 +1,87 @@
+.container{
+    display: flex;
+    background-color: lightblue;
+    padding: 15px;
+    box-shadow: 0px 3px whitesmoke;
+}
+#logo{
+    font-size: 55px;
+    margin-left: 45px;
+}
+.a{
+    justify-content: flex-end;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.a1{
+    padding: 0px 15px;
+    font-size: 25px;
```

Comment:
what is a, a1? please use names that are understood easily

Reply:
allirght